src/parameter/ParameterWidget.cc

```diff
@@ -292,9 +293,9 @@ void ParameterWidget::rebuildGroupMap(){
 			it++;
 		}
 	}
-	for (group_map::iterator it = groupMap.begin(); it != groupMap.end(); it++){
-		it->second.parameterVector.clear();
-		it->second.inList=false;
+	for (auto &it : groupMap) {
```

Comment:
`it` isn't an iterator. This time it's a "group map element". Unfortunately this is still a bit annoying until C++17 hits, so just renaming from `it` to `elem` or smth. will make this clearer.